charts/openebs/templates/kyverno/allow-capabilities.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if Values.rbac.kyverno Enabled }}
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: allow-add-capabilities
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+          Provides a list of capabilities that are allowed to be added to a container.
+spec:
+  validationFailureAction: audit
```

Comment:
We have to use `enforce` instead of `audit` in all places.

validationFailureAction controls admission control behaviors, when a policy rule fails:
 - use 'enforce' to block resource creation or modification
 - use 'audit' to allow resource updates and report policy violations